code/game/machinery/doors/firedoor.dm

```diff
@@ -23,12 +23,89 @@
 	interaction_flags_machine = INTERACT_MACHINE_WIRES_IF_OPEN | INTERACT_MACHINE_ALLOW_SILICON | INTERACT_MACHINE_OPEN_SILICON | INTERACT_MACHINE_REQUIRES_SILICON | INTERACT_MACHINE_OPEN
 	var/nextstate = null
 	var/boltslocked = TRUE
-	var/list/affecting_areas
 	var/being_held_open = FALSE
+	/// The merger id used to create/get the merger group in charge of handling tanks that share an internal gas storage
+	var/merger_id = "firelock"
+	/// The typecache of types which are allowed to merge internal storage
+	var/static/list/merger_typecache
+	///stores the merger datum if we are the owner
+	var/datum/merger/our_merger
```

Comment:
I don't know if merger members are meant to hold a reference to it rather than using `GetMergeGroup()` when needed. It doesn't seem like it IMO, but I just learned about the system now. @ninjanomnom could probably say otherwise.

Reply:
I'd go ahead and change it, since you only actually use it in one proc anyway.

Reply:
Will do that